man/docker-cp.1.md

```diff
@@ -28,12 +28,12 @@ supplying the initial forward slash is optional; The command sees
 `compassionate_darwin:tmp/foo/myfile.txt` as identical. If a `LOCALPATH` value
 is not absolute, is it considered relative to the current working directory.
 
-Behavior is similar to the common Unix utility `cp -a` in that directories are
-copied recursively with permissions preserved if possible. Ownership is set to
-the user and primary group on the receiving end of the transfer. For example,
-files copied to a container will be created with `UID:GID` of the root user.
-Files copied to the local machine will be created with the `UID:GID` of the
-user which invoked the `docker cp` command.
+Behavior is similar to the common Unix utility `cp -a` (unless `-L` option is
```

Comment:
@WeiZhang555 the man page should match whatever we decide to do with the `cp` reference